src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST" onsubmit="onSubmit()"
+    >
+      <fieldset class="groupedInfo">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="userSurname">Surname:</label>
+          <input
+            type="text"
+            name="userSurname"
+            id="userSurname"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="userName">Name:</label>
+          <input
+            type="text"
+            name="userName"
+            id="userName"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="yearsOld">How old are You?</label>
+          <input
+            type="number"
+            name="yearsOld"
+            id="yearsOld"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="birthDate">Full date of birth:</label>
+          <input
+            type="date"
+            name="birthDate"
+            id="birthDate"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="checkbox1">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="checkbox1"
+            id="checkbox1"
+            required
+          >
+        </div>
+      </fieldset>
+
+
+      <fieldset class="groupedInfo">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="userEmail">E-mail:</label>
+          <input
+            type="email"
+            name="userEmail"
+            id="userEmail"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="userpassword">Password:</label>
+          <input
+            type="password"
+            name="userpassword"
+            id="userpassword"
+            minlength="8"
+            maxlength="32"
+            required
+          >
```

Comment:
Use camelCase for id and class names.
```suggestion
          <label for="userPassword">Password:</label>
          <input
            type="password"
            name="userPassword"
            id="userPassword"
            minlength="8"
            maxlength="32"
            required
          >
```